service/src/main/java/bio/terra/cbas/controllers/RunSetsApiController.java

```diff
@@ -313,16 +339,13 @@ public static String getSubmissionUrl(
     };
   }
 
-  public static void captureRequestMetrics(RunSetRequest request) {
-    recordInputsInRequest(request.getWorkflowInputDefinitions().size());
-    recordOutputsInRequest(request.getWorkflowOutputDefinitions().size());
-    recordRecordsInRequest(request.getWdsRecords().getRecordIds().size());
-  }
-
-  public static void captureResponseMetrics(RunSetStateResponse response) {
```

Comment:
Since this was capturing how many Runs have been successfully submitted to Cromwell, I moved this to RunSetsService.

Reply:
awesome - we should continue looking for opportunities to move logic out of the controllers and into services!